app/assets/javascripts/search.js.coffee.erb

```diff
@@ -0,0 +1,28 @@
+class @Search
```

Comment:
Why is this file an ```.erb```?  There's no interpolation in it.  You also don't need ```.js.coffee```, just ```.coffee``` will be fine.